tests/samplers_tests/tpe_tests/test_parzen_estimator.py

```diff
@@ -222,25 +200,25 @@ def test_calculate_shape_check(
     mpe = _ParzenEstimator(
         {"a": mus}, {"a": distributions.FloatDistribution(-1.0, 1.0)}, parameters
     )
-    assert len(mpe._mixture_distribution.weights) == max(len(mus) + int(prior), 1)
+    assert len(mpe._mixture_distribution.weights) == max(
+        len(mus) + 1, 1
```

Comment:
We don't need the max operator.
```suggestion
        len(mus) + 1
```